libs/platform/auth/src/components/button/button.def.ts

```diff
@@ -0,0 +1,15 @@
+import { componentDef } from '@spryker-oryx/core';
+import { AuthButtonOptions } from './button.model';
+
+declare global {
+  interface FeatureOptions {
+    'oryx-auth-button'?: AuthButtonOptions;
+  }
+}
+
+export const authButtonComponent = componentDef({
```

Comment:
Let's rename it to `LoginButtonComponent`.

Reply:
But it doing both (login|logout) depends on auth state

Reply:
yup, we know :)

Reply:
I also prefer `LoginButtonComponent`, we could have an option on it to allow for also supporting the logout behaviour. I just don't like `AuthButtonComponent`, as you cannot grasp what it does (register? password related stuff? etc).

Reply:
Should we rename to `LoginTriggerComponent` to not bake in the UI type of this component?